lib/Common/Memory/Recycler.cpp

```diff
@@ -5240,6 +5240,7 @@ Recycler::FinishConcurrentCollect(CollectionFlags flags)
 
 #ifdef PROFILE_EXEC
     Js::Phase concurrentPhase = Js::ConcurrentCollectPhase;
+    static_cast<Js::Phase>(concurrentPhase);
```

Comment:
As -profile is disabled for GC phases, this is workaround for the unreferenced local variable. I have plan to enable -profile for GC phases so this is temporary.

---
In reply to: [66280866](https://github.com/Microsoft/ChakraCore/pull/1096#discussion_r66280866) [](ancestors = 66280866)